### PR TITLE
[Review] update vagrant readme with restart instructions change ansible openwhisk.yml

### LINKED ISSUE #596: Provide steps to re-deploy after restarting Vagrant VM
problem description
--------------------------
For Hello case 
./wsk action invoke /whisk.system/samples/echo -p message hello --blocking --result

returns   "error: [Errno 111] Connection refused"

steps to reproduce
--------------------------
1. Clone openwhisk
git clone --depth=1 https://github.com/openwhisk/openwhisk.git

2. Change directory to tools/vagrant/simple
cd openwhisk/tools/vagrant/simple

3. Run script to create vm and run hello action
vagrant up 
./wsk action invoke /whisk.system/samples/echo -p message hello --blocking --result

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #596

### CODE REVIEW

**Comment by csantanapr** [comment]
@rabbah please give it a try and merge

File: ansible/README.md [hunk 1]
```diff
@@ -142,12 +142,13 @@ cd <openwhisk_home>
 gradle distDocker
 cd ansible
 ansible-playbook -i environments/<environment> initdb.yml
-ansible-playbook -i environments/<environment> openwhisk.yml
+ansible-playbook -i environments/<environment> openwhisk_fresh.yml
```
Comment (csantanapr) [inline ansible/README.md]: @markusthoemmes ball of eyes here on the new `_fresh.yml`

File: tools/travis/build.sh [hunk 2]
```diff
@@ -24,7 +24,7 @@ cd $ROOTDIR
 
 cd $ROOTDIR/ansible
 
-$ANSIBLE_CMD openwhisk.yml
+$ANSIBLE_CMD openwhisk_fresh.yml
```
Comment (csantanapr) [inline tools/travis/build.sh]: @domdom82 @lehoanganh @markusthoemmes this is the only play to change the usage of "openwhisk.yml" ?

**Comment by lehoanganh** [comment]
I'm not sure about the usage of `openwhisk_fresk.yml`?

**Comment by markusthoemmes** [comment]
Me neither, why not just run `wipe.yml` and `postdeploy.yml` seperately? Having too many different versions as compositions of `openwhisk.yml` will bring us to the same hell `ant` did.

**Comment by csantanapr** [comment]
@lehoanganh @markusthoemmes This was one way to solve the problem.
I agree with @markusthoemmes I think doing `wipe.yml` and `posdeploy.yml`  separately 
I will remove refresh and update README and travis to use postdeploy.yml after running openwhisk.yml

FYI @rabbah you will need to adjust `redo` because this two steps will not be part of openwhisk.yml

**Comment by rabbah** [comment]
@csantanapr change line 199 of `redo` to

                  steps = 'setup, prereq, couchdb, initdb, wipe, deploy, catalog),

**Comment by csantanapr** [comment]
@rabbah review changes to redo, review and merge.
Any other changes please handle since I will be out. Thansk a bunch 💯

**Comment by rabbah** [comment]
@csantanapr will take care of it.

@lehoanganh:

`initdb` is a misleading name. we should rename it. it suggests that the db is initialized and ready to use after running this play book which is not the case; running `wipe` once is not optional (and should be `wipedb` at the very least).

`initdb` -> installs `subjects` view and default keys iff db doesn't already exist
`wipedb` -> drops the main database, recreates it, and installs required views

so `init-subject-db` is more descriptive and so is `init-maind-db`. can we do better? what do you think about all of this

**Comment by lfzCarlosC** [comment]
any update?

**Comment by rabbah** [comment]
Rebased and merged as https://github.com/openwhisk/openwhisk/commit/33e5f236dca9161db7bf5ef1504bdd8552c2f01b.

**Comment by csantanapr** [comment]
@lfzCarlosC this is now addressed. 
Please give it a try and provide feedback

**Pull request closed without merging**